docs/general/testing/upgrades.md

```diff
@@ -0,0 +1,184 @@
+
+1. In your Jellyfin server instance, navigate to the Dashboard -> Plugins -> Repositories.
+
+1. Delete the "Unstable" repository by clicking the trashcan button.
+
+1. Add a new repository with the "+" button. Name the repository whatever you wish ("Stable" is the default), and use the following as the Repository URL.
+
+   ```sh
+   https://repo.jellyfin.org/files/plugin/manifest.json
+   ```
+
+### Debian/Ubuntu (Unstable to Release)
+
+1. Edit your `/etc/apt/sources.list.d/jellyfin.sources` and remove `unstable` from the `Components:` line.
+
+   ```sh
+   ...
+   Components: main
+   ...
+   ```
+
+1. Run `sudo apt-get update`.
+
+1. Run `sudo apt-get install --reinstall jellyfin jellyfin-server jellyfin-web`. This will forcibly reinstall the new Stable version over top of the Unstable version. Jellyfin will automatically restart.
+
+### Docker (Unstable to Release)
+
+1. Stop your Jellyfin container.
+
+1. Pull the `jellyfin/jellyfin:latest` image tag.
+
+1. Start a new Jellyfin container with the updated image.
+
+### Post-Install (Unstable to Release)
+
+1. Once updated plugins become available, Jellyfin should automatically upgrade these to the next stable version as mentioned above. If you find any that do not, try to upgrade them manually.
+
+## Downgrading from Unstable to an older Stable
+
+As mentioned above, this process requires restoring from a backup taken before you first upgraded to Unstable.
+
+The exact details of this process depend on your platform and installation method. This document will detail the two most popular: Debian/Ubuntu packages via our repository, and Docker. The process is generally the same for others (e.g. manual downloads, Windows installers, etc.) with the exact details changed; for the specifics of individual platforms, please see the child articles under ["Testing Jellyfin Server"](/docs/general/testing/server).
+
+We assume you are already running Jellyfin Unstable releases.
+
+### Debian/Ubuntu (Unstable to Old Stable)
+
+1. Stop Jellyfin using the service manager e.g. `sudo service jellyfin stop` or `sudo systemctl stop jellyfin`. Ensure Jellyfin has actually stopped.
+
+1. Restore your backup of the configuration and data directories. Ensure you remove the current contents entirely (or move it out of the way) first.
+
+1. Edit your `/etc/apt/sources.list.d/jellyfin.sources` and remove `unstable` from the `Components:` line.
+
+   ```sh
+   ...
+   Components: main
+   ...
+   ```
+
+1. Run `sudo apt-get update`.
+
+1. Run `sudo apt-get install --reinstall jellyfin jellyfin-server jellyfin-web`. This will forcibly reinstall the new Stable version over top of the Unstable version. Jellyfin should automatically start; if not, use the service manager e.g. `sudo service jellyfin start` or `sudo systemctl start jellyfin`.
```

Comment:
```suggestion
1. Run `sudo apt-get install --reinstall jellyfin jellyfin-server jellyfin-web`. This will forcibly reinstall the new Stable version over top of the Unstable version. Jellyfin should automatically start; if not, `sudo service jellyfin start` or `sudo systemctl start jellyfin` to start Jellyfin.
```
This command should be the same on all debian / ubuntu distros except the crazy ones that remove systemd

Reply:
How about:

>use the service manager to start Jellyfin (`sudo service jellyfin start` or similar).

Reply:
that should work too